flink-connector-dynamodb/pom.xml

```diff
@@ -33,6 +33,15 @@ under the License.
     <artifactId>flink-connector-dynamodb</artifactId>
     <name>Flink : Connectors : Amazon DynamoDB</name>
 
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-test-utils</artifactId>
+			<version>1.16-SNAPSHOT</version>
```

Comment:
Why are we depending on a snapshot version and not the parent Flink version? Can this dependency not be managed from the dynamodb parent pom?

Reply:
Had offline discussion with Danny, we will use 1.16-SNAPSHOT version as there is no released version for flink-test-utils and flink-ci-tools yet. 

Todo: Once 1.16 is release, we will upgrade the version of dependencies in here to released verison